src/main/preset-data/package.groovy

```diff
@@ -21,21 +21,21 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-import org.apache.tools.ant.taskdefs.Zip;
-import org.apache.tools.ant.Project;
+import org.apache.tools.ant.taskdefs.Zip
+import org.apache.tools.ant.Project
 
-def baseDir = project.basedir;
+def baseDir = project.basedir
 
-File targetDir = new File(baseDir,"target/classes");
-targetDir.mkdirs();
+def targetDir = new File(baseDir, "target/classes")
+targetDir.mkdirs()
 
-File dataSetRoot = new File(baseDir,"src/main/preset-data");
+def dataSetRoot = new File(baseDir, "src/main/preset-data")
```

Comment:
This definitely looks like a change in the wrong direction to me.

Reply:
You refert to `def` instead of `File`?
That's reducing type duplication. 
Groovy is a dynamic language.
Having types everywhere doesn't provide much more confidence as a lot of things are just checked as runtime unless `@CompileStatic` is used.
But I can revert this if really desired.
Just let me know.

Reply:
I'm not sure on the precise context, but even in Java, this is idiomatic now. For example:

```java
var f = new File(...)
```

versus:

```java
File f = getRoot();
```

Essentially, you should have an obvious type in the line before using `def`/`var`.